### PR TITLE
Added in memory test storage

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: germanium/storage.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
+import os
```
Comment (matllubos) [inline germanium/storage.py:1]: most of it is copy of dj-inmemorystorage (https://pypi.org/project/dj-inmemorystorage/) But I must fix bugs a improve it for testing purposes

File: germanium/storage.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    filesystem_name = 'default'
+
+    def __init__(self, base_url=None):
+        if base_url is None:
+            base_url = settings.MEDIA_URL
+        self.base_url = base_url
+
+    @property
+    def filesystem(self):
+        return test_filesystems[os.getpid()][self.filesystem_name]
```
Comment (matllubos) [inline germanium/storage.py:172]: see the os.getpid() it is necessary for testing in multiple processes

File: germanium/test_cases/default.py [hunk 3]
```diff
@@ -41,7 +43,8 @@ def tearDown(self):
         self.tear_down()
 
     def tear_down(self):
-        pass
+        if os.getpid() in test_filesystems:
+            del test_filesystems[os.getpid()]
```
Comment (matllubos) [inline germanium/test_cases/default.py:47]: automatically clean storage after test completition

File: germanium/storage.py [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+from django.utils import timezone
+
+from collections import defaultdict
+
+
+class PathDoesNotExist(Exception):
+    pass
+
+
+class InMemoryNode(object):
```
Comment (Formulka) [inline germanium/storage.py:23]: not necessary to inherit from object

**Review by Formulka — APPROVED** [review]

**Pull request merged**